app/assets/styles/_react-select.scss

```diff
@@ -53,7 +53,8 @@ $selectDropdownPaddingRight: 4px;
     font-size: $selectFontSize;
     color: $black;
     cursor: pointer;
-
+    padding: 0;
```

Comment:
Was this causing the dropdown height to change when value was selected?

Reply:
Yes, the clear icon was 20px + 8x2 that bigger than the largest item in the select box, which push it ~2px. 

By removing this, the highest item decide the height of the select is the dropdown icon